src/modules/Header/components/Header.js

```diff
@@ -26,23 +30,47 @@ const HeaderBarUpper = styled.div`
   width: 100%;
   background: ${COLORS.WHITE};
   max-height: 50px;
-
 `
 
 const HeaderBarLower = styled.div`
   display: flex;
   justify-content: ${props => props.justifyContent ? props.justifyContent : 'flex-start'};
   margin: 0 auto;
-  padding-left:20px;
   width: 100%;
   background: ${COLORS.OIVA_MENU_BG_COLOR};
   max-height: 50px;
+  overflow: hidden;
+  @media ${MEDIA_QUERIES.MOBILE} {
+    padding-left:32px;
+    flex-flow: column;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 0;
+    height: 40px;
+    opacity: 0.99;
+    z-index: 100;
+
+    &:before {
+      content: "Ξ";
+      margin-left: -20px;
+      margin-top: 10px;
```

Comment:
Tailwind provides a way to define margins in rem units. For example mt-4. We should use pixels. They don't scale.